docs/tutorials/hyp3_isce2_burst_stack_for_ts_analysis.ipynb

```diff
@@ -0,0 +1,467 @@
+    "\n",
+    "stack_start = parse_date('2014-05-01 00:00:00Z')\n",
+    "stack_end = parse_date('2015-05-01 00:00:00Z')\n",
+    "max_temporal_baseline =  37 #days\n",
+    "\n",
+    "data_dir.mkdir(parents=True, exist_ok=True)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0e62c544",
+   "metadata": {},
+   "source": [
+    "## 1. Select InSAR pairs with ASF Search"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4498c66c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "import asf_search as asf\n",
+    "import pandas as pd\n",
+    "\n",
+    "search_results = asf.search(\n",
+    "        platform=asf.SENTINEL1,\n",
+    "        intersectsWith='POLYGON((-135.7684 57.0473,-135.7389 57.0473,-135.7389 57.0583,-135.7684 57.0583,-135.7684 57.0473))',\n",
+    "        start='2014-05-01',\n",
+    "        end='2015-05-01',\n",
+    "        processingLevel=asf.BURST,\n",
+    "        beamMode=asf.IW,\n",
+    "        flightDirection=asf.DESCENDING,\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9f7ccdcc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "baseline_results = asf.baseline_search.stack_from_product(search_results[-1])\n",
+    "\n",
+    "columns = list(baseline_results[0].properties.keys()) + ['geometry', ]\n",
+    "data = [list(scene.properties.values()) + [scene.geometry, ] for scene in baseline_results]\n",
+    "\n",
+    "stack = pd.DataFrame(data, columns=columns)\n",
+    "stack['startTime'] = stack.startTime.apply(parse_date)\n",
+    "\n",
+    "stack = stack.loc[(stack_start <= stack.startTime) & (stack.startTime <= stack_end)]\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e8d9f38b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "sbas_pairs = set()\n",
+    "\n",
+    "for reference, rt in stack.loc[::-1, ['sceneName', 'temporalBaseline']].itertuples(index=False):\n",
+    "    secondaries = stack.loc[\n",
+    "        (stack.sceneName != reference)\n",
+    "        & (stack.temporalBaseline - rt <= max_temporal_baseline)\n",
+    "        & (stack.temporalBaseline - rt > 0)\n",
+    "    ]\n",
+    "    for secondary in secondaries.sceneName:\n",
+    "        sbas_pairs.add((reference, secondary))\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9e5c5b0b",
+   "metadata": {},
+   "source": [
+    "## 2. Request On Demand InSAR products from ASF HyP3\n",
+    "\n",
+    "Use your [NASA Earthdata login](https://urs.earthdata.nasa.gov/) to connect to [ASF HyP3](https://hyp3-docs.asf.alaska.edu/)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "be78f415",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "import hyp3_sdk as sdk\n",
+    "\n",
+    "hyp3 = sdk.HyP3('https://hyp3-enterprise-test.asf.alaska.edu', prompt=True)"
```

Comment:
We also support InSAR burst jobs via `hyp3-test`, so that might be the more appropriate API to use. But this won't work at all until we support burst jobs via the SDK.
```suggestion
    "hyp3 = sdk.HyP3('https://hyp3-test-api.asf.alaska.edu', prompt=True)"
```